util/shared/descriptions.test.mjs

```diff
@@ -16,18 +16,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * This module sets up the mock transport layer immediately, instead of letting the SDK wait 500ms
- */
+import { getMarkdown, getAllMarkdownNames } from './descriptions.mjs'
```

Comment:
## And here are some tests

The presence of these will allow `npm test` to have something to do.